src/index.js

```diff
@@ -30,6 +31,11 @@ function formatter(values, param, type, onlyIfExists) {
     return param.name;
   }
 
+  // Handle file uploads. Specifically arrays of file uploads which need to be formatted very specifically.
+  if (param.schema && param.schema.type === 'array' && param.schema.items.format === 'binary') {
```

Comment:
Since we have a lot of specs in the DB that don't have `type` or` items` for arrays, and we have handling in `getParametersAsJsonSchema()` to rectify this out of extreme caution you might want to optionally chain this.

```suggestion
  if (param.schema && param.schema?.type === 'array' && param.schema?.items?.format === 'binary') {
```